### ISSUE TITLE
Why not use pre_normalization() function?

### ISSUE THREAD

**Issue by lichengxian** [yfsong0709/ResGCNv1#16]
When I evaluate pretrained model on ntu-sub dataset, I found NTU_Feeder input the raw 3d skeleton points in the net, but in the paper, fig.2 show it use BatchNorm, Why not use pre_normalization() function?
